docs/manual-EN/1.overview/2.quick-start/installing-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,157 @@
+# Installing Nebula Graph with Docker-compose
+
+Before you can install **Nebula Graph**, you must ensure that `Docker Engine` and `Docker Compose` are installed.
+
+## Requirements
+
+Before you install **Nebula Graph**, you must ensure that the following software and hardware requirements are met.
+
+### Software Requirements
+
+Ensure that your system is one of the following:
+
+* Fedora 29, 30
+* Centos, 6.5, 7.5
+* Ubuntu 16.04, 18.04
+
+### Hardware Requirements
+
+Ensure that your disk space and memory meet the following requirements:
+
+* 30G for disk space
+* 8G for memory
+
+## Installing Docker Engine
+
+In this section, we will show you how to install the latest version of `Docker Engine for Community`.
+
+You can install the latest version of `Docker Engine` by the following steps:
+
+1. Uninstall the old versions of `Docker`.
+
+```$
+sudo apt-get remove docker docker-engine docker.io containerd runc
+```
+
+2. Update the `apt` package index.
+
+```$
+sudo apt-get update
+```
+
+3. Install packages to allow `apt` to use a repository over HTTPS.
+
+``` $
+sudo apt-get install \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg-agent \
+    software-properties-common
+    ```
+4. Add Docker’s official GPG key:
+
+```$
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+```
+
+5. Verify that you now have the key with the fingerprint `9DC8 5822 9FC7 DD38 854A E2D8 8D81 803C 0EBF CD88`, by searching for the last 8 characters of the fingerprint.
+
+```$
+sudo apt-key fingerprint 0EBFCD88
+```
+
+6. Use the following command to set up the `stable` repository.
+
+```$
+sudo add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
+```
+
+7. Update the `apt` package index.
+
+```$
+sudo apt-get update
+```
+
+8. Install the latest version of `Docker` and `containerd`.
+
+```$
+sudo apt-get install docker-ce docker-ce-cli containerd.io
+```
+
+9. Verify that `Docker` is installed correctly by running the `hello-world` image.
+
+```$
+sudo docker run hello-world
+```
+
+**Note**: 
+
+1. After entering the `sudo docker run hello-world` command, if the `Hello from Docker` message is displayed, it indicates that `Docker Engine` is installed successfully.
+
+2. For details about installing `Docker Engine - Community`, you can refer to https://docs.docker.com/install/linux/docker-ce/ubuntu/.
```

Comment:
what if the user is not using ubuntu? we'd better use this link here 
https://docs.docker.com/v17.12/install/

Reply:
I don't think we need so many steps to install docker, please help review this part. @whitewum @yixinglu